creator-node/src/services/stateMachineManager/stateMachineUtils.js

```diff
@@ -126,7 +126,31 @@ const retrieveClockValueForUserFromReplica = async (replica, wallet) => {
   return clockValue
 }
 
+const processJob = async (
+  jobName,
+  job,
+  jobProcessor,
+  parentLogger,
+  defaultResult = {}
+) => {
+  const { id: jobId, data: jobData } = job
+
+  const jobLogger = createChildLogger(parentLogger, { jobName, jobId })
+  jobLogger.info(`New job: ${JSON.stringify(job)}`)
```

Comment:
curious what the logger metadata ends up looking like here

Reply:
locally it just adds it in parentheses at the end, so for this particular log it shows:
```
New job: {"id":"1","name":"monitor-state","data":{"lastProcessedUserId":0,"discoveryNodeEndpoint":"http://dn1_web-server_1:5000"},"opts":{"removeOnComplete":20,"removeOnFail":20,"delay":30000,"attempts":1,"timestamp":1654645727592},"progress":0,"delay":0,"timestamp":1654645727592,"attemptsMade":0,"stacktrace":[],"returnvalue":null,"finishedOn":null,"processedOn":1654645757593} (queue=state-monitoring-queue, jobName=monitor-state, jobId=1, logLevel=info)
```

Reply:
sweet

Reply:
i guess my question was really - what do nested child loggers look like, but thats a more general question and one that i can just answer myself haha